File: despatch.WEB/app/src/swagger/models/user-phone-numbers-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { PhoneNumberResponse } from './phone-number-response';
import { UserResponse } from './user-response';
export interface UserPhoneNumbersResponse extends BaseEntityResponse{
  phoneNumber?: PhoneNumberResponse;
  phoneNumberId?: string;
  user?: UserResponse;
  userId?: string;
}

File: despatch.Domain/Models/Default/Response/UserPhoneNumbersResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserPhoneNumbersResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string PhoneNumberId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual PhoneNumberResponse PhoneNumber { get; set; }
    }
}
